feature/profile/src/main/java/com/teamwable/profile/ProfileDeleteReasonFragment.kt

```diff
@@ -0,0 +1,68 @@
+package com.teamwable.profile
+
+import androidx.navigation.fragment.findNavController
+import com.teamwable.profile.databinding.FragmentProfileDeleteReasonBinding
+import com.teamwable.ui.base.BindingFragment
+
+class ProfileDeleteReasonFragment : BindingFragment<FragmentProfileDeleteReasonBinding>(FragmentProfileDeleteReasonBinding::inflate) {
+    private val checkBoxList by lazy {
+        with(binding) {
+            listOf(
+                cbProfileDeleteReasonCheck1,
+                cbProfileDeleteReasonCheck2,
+                cbProfileDeleteReasonCheck3,
+                cbProfileDeleteReasonCheck4,
+                cbProfileDeleteReasonCheck5,
+                cbProfileDeleteReasonCheck6,
+                cbProfileDeleteReasonCheck7
+            )
+        }
+    }
```

Comment:
p1 : 
```suggestion
    private lateinit var checkBoxList: List<CheckBox>
```
by lazy를 사용하게 되면 이전 객체에 대한 참조가 남아있게 됩니다!
fragment view의 경우 onDestoryView에서 사라지지만
fragment 객체의 경우 onDestroy시에 사라지므로 다음 화면 갔다가 오게 되면 fragment객체는 살아있는 상태입니다.
따라서 checkboxList가 초기화가 안되서 아래 로직들이 실행 안되는 거 같습니다!

Reply:
맞아요 분명 lifecycle 관련해서 문제가 있다고 생각했는데..! 그런데 돈비 코드 참고한 것인데 돈비 앱에서 알맞게 작동하고 있는지 궁금하네용(회원가입 약관 동의)

Reply:
돈비 앱에서 해당 부분은 액티비티라 상관없습니다!